### PR TITLE
Bugfix updates to the utilities and version bumps 

### PR DESCRIPTION
**Pull request by ontheslab**
Hello Frederic,

My name is Mark and I've been working with Matt Boytim (https://github.com/maboytim) to try to pull together a number of bugfixes that have been made by several people and also to pull the version numbers into a better line. I understand there was some confusion some time back when the firmware would not compile. The changes are all listed in the code updates and new readme files. Tools will become 2.05 and Firmware will become 2.16

None of this includes the video updates etc that are in the works. All has been tested to ensue it compiles by more than one person.

Many thanks.

Mark Casteleijn

### CODE REVIEW

File: sw/z80/utils/import2core.z80 [hunk 1]
```diff
@@ -1,5 +1,11 @@
 ; Core - Shared by Versions for all OS's
 
+; Bug found by: hallorant
+; A paramater parsing bug was found in import2core.z80 that has been fixed.  The
```
Comment (fvecoven) [inline sw/z80/utils/import2core.z80:4]: typo: parameter

**Review by fvecoven — APPROVED** [review]
Thanks for these fixes and keeping the project alive !

File: sw/z80/utils/import2core.z80 (hunk 1 above)
Comment (ontheslab) [reply sw/z80/utils/import2core.z80:4]: Well spotted thank you! ;-) I just copied his comments over!

**Comment by ontheslab** [comment]
> Thanks for these fixes and keeping the project alive !

Many thanks for creating the device! Having so much fun with it in on the 4p!

**Comment by ontheslab** [comment]
Hello again Fred, (@veco) I hope you don't mind me addressing you so?

Just wondering if you are happy with the update, are going to merge them into the "main" branch or even create a new branch?

I must admit to being a tad - ignorant - of the  processes here, this not how we used to do it in the "old" days ;-)

Many thanks again, Mark (Intangybles)

**Comment by fvecoven** [comment]
Merged now. Sorry, I thought that by approving, you could merge yourself.

**Pull request merged**